### PR TITLE
Build pending nonprofit page

### LINKED ISSUE #27: 😎 Build pending nonprofit page
**Type**: client

**FOF**: 1

## Description
- We want to let nonprofit admins know that their account is pending while their registration details are being validated

## Screenshot
image

## Todo

- [ ] Build the pending registration screen!

## Links

- [Figma](https://www.figma.com/file/qELi7Tz3K4XxXQmCVrrpRE/Clear-Path-x-H4I?node-id=1%3A2&t=wnmmIblE2fpTrpNM-0)
- [Styled Components Tips](https://javascript.plainenglish.io/react-7-tricks-to-work-with-styled-components-20eb155934b1)

## Learnings
- Coding! 😎

### PR DESCRIPTION
**Pull request by szhuang21**
## Status:

:rocket: Ready

## Changes

- added arrow-right.svg, pending-check.svg
- created index.tsx, styles.tsx, and types.ts files for pending component 
- put pending component into https://clearpath-api.vercel.app/
- redirects to home clearpath website
- if info@clearpathnyc.org is clicked, it opens email

## Screenshots
image

### CODE REVIEW

File: src/components/pending/styles.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  position: relative;
+  height: 100%;
+  flex-grow: 1;
+
+  @media ${device.tablet} {
+    padding: 0 16px 16px;
+  }
+`;
+
+export const Heading = styled.h3`
```
Comment (ayanmallik) [inline src/components/pending/styles.tsx:18]: One thing I notice is that the heading isn't centered on mobile:
image

Nobody can escape my detective work! 🕵️ 🔎

**Review by ayanmallik — APPROVED** [review]
Sophia, this PR makes me a `:happy-capy:`! 😊 🐀

This PR is actually so clean 🧼 - might need you to hop back on dev work for the rest of the semester 🤗 😅 

Hope you had fun hitting the VS Code 👩‍💻 🤓 - this pending screen looks fantabulous ✨

Pro tip: run `yarn format` before pushing to make sure you pass the linter check ✅

As always, let me and Sophia (minus Sophia) know if you have any questions! Let's get this merged in soon 🚀

File: package.json [hunk 2]
```diff
@@ -35,6 +35,7 @@
   "scripts": {
     "start": "react-scripts start",
     "start:local": "REACT_APP_BASE_URL=http://localhost:8081 react-scripts start",
+    "start:live": "REACT_APP_BASE_URL=https://clearpath-api.vercel.app react-scripts start",
```
[the remark this answers is not in the document]
Comment (szhuang21) [reply package.json:38]: LMAO i dont think i know who youre talking ab...

File: src/components/pending/index.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+export const Pending = () => {
+  const email = "info@clearpathnyc.org";
+  return (
+    <div>
+      <ContentContainer>
+        <Check>
+          <PendingCheckIcon />
+        </Check>
+        <Heading>Your Request is Pending!</Heading>
+        <ReturnText href="https://www.clearpathnyc.org/">
```
[the remark this answers is not in the document]
Comment (szhuang21) [reply src/components/pending/index.tsx:15]: wait i thought this link was supposed to take the people to the clearpath org page tho like the home page?

File: src/components/pending/styles.tsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+`;
+
+export const ReturnText = styled.a`
+  font-size: 24px;
+  font-weight: 100;
+  line-height: 24px;
+  display: flex;
+  justify-content: center;
+  padding-bottom: 15px;
+  color: ${COLORS.textPrimary};
```
Comment (ajain1921) [inline src/components/pending/styles.tsx:40]: I think with the way they're currently managing global theme this should be ok but just for consistency we can follow what they do in other components: `${(p) => p.theme.colors...}`.

**Pull request merged**